src/strands/tools/mcp/mcp_client.py

```diff
@@ -63,17 +63,23 @@ class MCPClient:
     from MCP tools, it will be returned as the last item in the content array of the ToolResult.
     """
 
-    def __init__(self, transport_callable: Callable[[], MCPTransport]):
+    def __init__(self, transport_callable: Callable[[], MCPTransport], server_startup_timeout: int = 30):
```

Comment:
We reviewed this internally - let's name this `startup_timeout` and make it non-positional.

```suggestion
    def __init__(self, transport_callable: Callable[[], MCPTransport], *, startup_timeout: int = 30):
```